week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,24 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+//Mart.random() returns a random number between 0 (inclusive),  and 1 (exclusive).
+// getNumber() returns a random integer from 0 to 9
 function getNumber() {
     return Math.random() * 10;
 }
 
+
 // Add comments to explain what this function does. You're meant to use Google!
+
+//The concat() method is used to join two or more strings.
```

Comment:
whatever related to concat() is inside the "s" function, 
so, it good practice to such comment inside the function's body and above the line there where concat() is used, or at the end of that line (less common practice).